metagraph/src/cli/build.cpp

```diff
@@ -249,7 +250,15 @@ int build_graph(Config *config) {
                                                              get_verbose()));
         }
 
-    } else {
+    } else if (config->graph_type == Config::GraphType::SSHASH){
+
+        graph.reset(new DBGSSHash(files.at(0), config->k, config->graph_mode));
```

Comment:
nit: `files[0]`? Also `graph = std::make_unique<DBGSSHash>(...)` seems to be the [preferred approach](https://stackoverflow.com/a/41247901/4628730), but maybe not that important, given that we use `graph.reset(new ...)` everywhere throughout the file.

Reply:
We're using `.at` so that it would throw an exception if there are no files passed in. In the end, this is only a single call.